lib/input.js

```diff
@@ -103,7 +104,12 @@ export default class Input {
     if (json.hash) {
       const input = new Input(Outpoint.fromJSON(json));
       if (json.signer) {
-        input.setSig(json.r, json.s, json.v, json.signer);
+        input.setSig(
+          new Buffer(json.r.replace('0x', ''), 'hex'),
```

Comment:
Also `toBuffer` from `ethereumjs-util` can be used for this operation. Like this: `toBuffer(json.r)`

https://github.com/ethereumjs/ethereumjs-util/blob/master/docs/index.md#tobuffer

Reply:
Thanks!